### DISCUSSION TITLE
Shields Gazette Article

### DISCUSSION THREAD

**Discussion by AlexHedley** [sanddancer-jpg/sanddancer-jpg.github.io#4] [Show and tell]
https://www.shieldsgazette.com/news/englands-first-official-japan-park-golf-course-opens-in-south-shields-north-marine-park-4745743

**Comment by sanddancer-jpg** [comment]
Cheers Alex, catch up with you soon
Many thanks,
Derek
Sent from my iPhone

On 18 Aug 2024, at 19:15, Alex Hedley ***@***.***> wrote:

﻿

https://www.shieldsgazette.com/news/englands-first-official-japan-park-golf-course-opens-in-south-shields-north-marine-park-4745743
